### ISSUE TITLE
Subdomains don't work on multisite

### ISSUE THREAD

**Issue by dashaluna** [Chassis/Chassis#429]
**Issue**
I've setup chassis with the following `chassis.local.yaml` file (below).

1. I create a subdomain ex `test.unison-branches.local`
2. Go to URL `test.unison-branches.local` and get 
```
This site can’t be reached

test.unison-branches.local’s server DNS address could not be found.
Search Google for test unison branches local
ERR_NAME_NOT_RESOLVED
```
3. `vagrant provision` doesn't solve the issue
4. following `vagrant reload` doesn't solve the issue

I've tried adding `test.unison-branches.local` to the `chassis.local.yaml` file. Then:
1. I run `vagrant provision` - the `test.unison-branches.local` URL seems to work.
2. If I run a following `vagrant reload` command, it's back to not working.

As I understand from docs: http://docs.chassis.io/en/latest/config/#multisite Specifying the following in `yaml` file
```
hosts:
 - unison-branches.local
multisite: subdomains
```
should make sure that creating and using new subdomains should just work - without needing to add that new subdomain to `yaml` file and the following `vagrant provision`.

**My setup**
1. What operating system do you use? => **macOS Sierra 10.12.6**

2. What version of Vagrant are you running? => **Vagrant 1.9.2**

3. Are you using VirtualBox or VMWare and which version are you using? => **VM 5.1.14** 

4. Do you have a custom YAML file? If so, what does it contain?
```
hosts:
 - unison-branches.local
multisite: subdomains
extensions:
 - Chassis/phpcs
 - Chassis/Xdebug
 - Chassis/db-backup
paths:
 # Path to your project directory relative the Chassis.
 base: unison-branches
 wp: wordpress
 content: content
database:
 name: wordpress
 user: wordpress
 password: vagrantpassword
 prefix: wp_
tester_db:
 name: tests
 user: wordpress
 password: vagrantpassword
 prefix: test_
```
5. Do you have any Chassis extensions installed? If so, what are they?
As above in the yaml file

**Comment by nathanielks** [comment]
I have the same issue!

**Comment by joemcgill** [comment]
Could be a duplicate of #472?

**Comment by roborourke** [comment]
I'm not sure this was ever supported out of the box, I've always had to specify my subdomains eg:

```yaml
hosts:
  - example.local
  - foo.example.local
  - bar.example.local
```

Would be nice to support wildcard out of the box but not sure if it's possible.

**Comment by BronsonQuick** [comment]
@roborourke Yeah we used to handle that with the Vagrant Hostsupdater plugin then we removed support for it as we went down the Avahi route but as I'd sure you're aware subdomains randomly drop out on Avahi.

**Comment by rdlmda** [comment]
Then please update your documentation, because currently it states: 

"Subdirectory mode is great, but subdomain mode is even better. With subdomains, a site called “test” would be available at test.vagrant.local. This is one of the most common ways that multisite is set up, since it also means you’re separating your sites more cleanly. Simply set the value to subdomains and reprovision. **Once this is done, subdomains will work automatically in your browser**. Create and remove sites at will, and Chassis will ensure it just works."
